lib/parse.js

```diff
@@ -10,6 +10,12 @@ var DOMParser = require('xmldom').DOMParser;
 
 exports.parse = parse;
 
+var NODE_TYPES = {
```

Comment:
yeah but the problem with that is it's more work to verify PRs are good *before* they're merged. I would rather not merge anything to master and have the tests fail after the fact.